MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/renderer/textureview/TextureViewRenderThread.java

```diff
@@ -390,7 +392,7 @@ boolean createSurface() {
 
       // Create an EGL surface we can render into.
       TextureView view = textureViewWeakRef.get();
-      if (view != null) {
+      if (view != null && view.getSurfaceTexture() != null) {
```

Comment:
LGTM, though noting that Android docs generally recommend using the listener instead to ensure the SurfaceTexture is not just present but also available: 

> A TextureView's SurfaceTexture can be obtained either by invoking `getSurfaceTexture()` or by using a `SurfaceTextureListener`. It is important to know that a SurfaceTexture is available only after the TextureView is attached to a window (and `onAttachedToWindow()` has been invoked.) It is therefore highly recommended you use a listener to be notified when the SurfaceTexture becomes available.

ref: https://developer.android.com/reference/android/view/TextureView

Reply:
We actually do use `SurfaceTextureListener` to obtain and destroy the `TextureSurface` in `TextureViewRenderThread`. The code above is being executed in the EGLHolder, which doesn't have access to wrapping TextureViewRenderThread. In this case we don't need the actual instance of SurfaceTexture, just validating state is enough here.